### PR TITLE
Disable the ability to view cancelled player commands

### PR DESCRIPTION
**Pull request by ghost**
This would mean that you cannot see the commands of persons who have been ICU controlled.
This would be a nice quality-of-life update because some bots who have been ICU controlled can still clog up CommandSpy even though their commands mean nothing.

### CODE REVIEW

**Comment by QuadraticKid** [comment]
yes do this

**Comment by Apmunute** [comment]
Yes.

File: src/main/java/pw/kaboom/commandspy/Main.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
 		for (String uuidString : config.getKeys(false)) {
```
Comment (mathiascode) [inline src/main/java/pw/kaboom/commandspy/Main.java:73]: Thanks! Could you move the check outside the loop?
```suggestion
		if (event.isCancelled()) {
			return;
		}

		for (String uuidString : config.getKeys(false)) {
```

Comment (ghost) [reply src/main/java/pw/kaboom/commandspy/Main.java:73]: Sure!

File: src/main/java/pw/kaboom/commandspy/Main.java [hunk 2]
```diff
@@ -66,10 +66,14 @@ public boolean onCommand(final CommandSender sender, final Command cmd, final St
...
 	void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
+		if (event.isCancelled()) {
+			return;
+		}
+
 		for (String uuidString : config.getKeys(false)) {
 			UUID uuid = UUID.fromString(uuidString);
 
-			if (Bukkit.getPlayer(uuid) != null) {
+			if (Bukkit.getPlayer(uuid) != null && !event.isCancelled()) {
```
Comment (mathiascode) [inline src/main/java/pw/kaboom/commandspy/Main.java]:
```suggestion
			if (Bukkit.getPlayer(uuid) != null) {
```

Comment (ghost) [reply src/main/java/pw/kaboom/commandspy/Main.java]: Are you moving the check?

Comment (ghost) [reply src/main/java/pw/kaboom/commandspy/Main.java]: Because that code would remove the functionality of this pull request altogether

Comment (mathiascode) [reply src/main/java/pw/kaboom/commandspy/Main.java]: Yes, this check is superfluous after the other one was added outside the loop.

**Review by mathiascode — APPROVED** [review]

**Pull request merged**